migration/idempotent/007-tracing-tags.sql

```diff
@@ -140,25 +147,27 @@ $do$;
 -- regexp not matches
 -------------------------------------------------------------------------------
 CREATE OR REPLACE FUNCTION _ps_trace.eval_regexp_not_matches(_op ps_tag.tag_op_regexp_not_matches)
-RETURNS jsonb[]
+    RETURNS jsonb[]
+    SET search_path = pg_catalog
 AS $func$
-    SELECT coalesce(array_agg(jsonb_build_object(a.key_id, a.id)), array[]::jsonb[])
+    SELECT coalesce(pg_catalog.array_agg(pg_catalog.jsonb_build_object(a.key_id, a.id)), array[]::jsonb[])
     FROM _ps_trace.tag a
-    WHERE a.key = _op.tag_key
+    WHERE a.key OPERATOR(pg_catalog.=) _op.tag_key
     -- if the jsonb value is a string, apply the regex directly
     -- otherwise, convert the value to a text representation, back to a jsonb string, and then apply
     AND CASE jsonb_typeof(a.value)
-        WHEN 'string' THEN jsonb_path_exists(a.value, format('$?(!(@ like_regex "%s"))', _op.value)::jsonpath)
-        ELSE jsonb_path_exists(to_jsonb(a.value#>>'{}'), format('$?(!(@ like_regex "%s"))', _op.value)::jsonpath)
+        WHEN 'string' THEN pg_catalog.jsonb_path_exists(a.value, pg_catalog.format('$?(!(@ like_regex "%s"))', _op.value)::jsonpath)
+        ELSE pg_catalog.jsonb_path_exists(to_jsonb(a.value #>> '{}'), pg_catalog.format('$?(!(@ like_regex "%s"))', _op.value)::jsonpath)
```

Comment:
the #>> operator needs to be schema qualified

Reply:
same as above